packages/@o3r/workspace/schematics/library/index.ts

```diff
@@ -46,7 +46,7 @@ function generateModuleFn(options: NgGenerateModuleSchema): Rule {
         inManifest: [
           {
             range: `${options.exactO3rVersion ? '' : '~'}${ownPackageJsonContent.version!}`,
-            types: [NodeDependencyType.Default]
+            types: [NodeDependencyType.Dev]
```

Comment:
Why not peerDep ?

Reply:
`setupDependencies` will add the package in the root package.json as well and we do not want to add peerDeps there.